### PR TITLE
BOAC-3321, proper SQL query wired in to admit-cohort.to_api_json

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-3321

For now, BOA assumes `admit_status`, `application_fee_waiver_flag`, `current_sir`, and 12 others to have SQL-type boolean in Nessie RDS. See https://jira.ets.berkeley.edu/jira/browse/NS-791 for the complete list. This PR has the cohort query and admit-student view expecting booleans. This code will evolve.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1581,23 +1581,23 @@
...
       }
     },
     "@fortawesome/free-solid-svg-icons": {
       "version": "5.12.1",
       "resolved": "https://registry.npmjs.org/@fortawesome/free-solid-svg-icons/-/free-solid-svg-icons-5.12.1.tgz",
       "integrity": "sha512-k3MwRFFUhyL4cuCJSaHDA0YNYMELDXX0h8JKtWYxO5XD3Dn+maXOMrVAAiNGooUyM2v/wz/TOaM0jxYVKeXX7g==",
       "requires": {
-        "@fortawesome/fontawesome-common-types": "0.2.27"
+        "@fortawesome/fontawesome-common-types": "^0.2.27"
       }
```
Comment (paulkerschen) [inline package-lock.json]: @johncrossman Did these package-lock.json changes stow away with the rest of the code?

Comment (johncrossman) [reply package-lock.json]: @paulkerschen - `npm install` after rebase made this change. I included it in PR assuming no harm. Want me to revert?

Comment (johncrossman) [reply package-lock.json]: A consequence of https://github.com/ets-berkeley-edu/boac/pull/2380, I assume.

Comment (paulkerschen) [reply package-lock.json]: @johncrossman No revert necessary, I think - just leery of that situation we sometimes get into where different people's computers handle `npm install` differently and the package details keep toggling back and forth. But let's see if this one stays quiet.

**Pull request merged**